Dockerfile

```diff
@@ -24,6 +24,7 @@ ENV CONTAINER 'docker'
 ENV PROGRESS true
 ENV COMMIT_SHA $commit_sha
 ENV CALYPSO_ENV production
+ENV NODE_ENV production
```

Comment:
FWIW, when we're running the production instance, we specify this env var as well, so I don't see why we shouldn't be doing it when building it, too.